lib/Benchmark/Runner.php

```diff
@@ -30,18 +31,21 @@ class Runner
     private $subjectBuilder;
     private $subjectMemoryTotal;
     private $subjectLastMemoryInclusive;
+    private $separateProcess;
 
     public function __construct(
         CollectionBuilder $finder,
         SubjectBuilder $subjectBuilder,
-        ProgressLogger $logger = null
+        ProgressLogger $logger = null,
+        $separateProcess = false
     ) {
         $this->logger = $logger ?: new NullProgressLogger();
         $this->finder = $finder;
         $this->subjectBuilder = $subjectBuilder;
+        $this->separateProcess = $separateProcess;
     }
 
-    public function runAll()
+    public function runAll($noSetup = false, $parameters = null)
```

Comment:
There is not really a reason to treat these parameters separately from `separateProcess`. They should either be moved to the constructor or vice-versa.